diff -Nuard gnomeradio-1.4.orig/Makefile.am gnomeradio-1.4/Makefile.am
--- gnomeradio-1.4.orig/Makefile.am	2002-10-18 21:28:06.000000000 +0200
+++ gnomeradio-1.4/Makefile.am	2003-12-14 17:03:56.122760184 +0100
@@ -5,7 +5,7 @@
 Applications_DATA = $(applications_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
 
-schemadir = @GCONF_SCHEMA_FILE_DIR@
+schemadir = $(GCONF_SCHEMA_FILE_DIR)
 schema_in_files = gnomeradio.schemas.in
 schema_DATA = $(schema_in_files:.schemas.in=.schemas)
 @INTLTOOL_SCHEMAS_RULE@
