diff -urN gnomeradio-1.4.orig/src/gui.c gnomeradio-1.4/src/gui.c
--- gnomeradio-1.4.orig/src/gui.c	Tue Dec 17 23:33:03 2002
+++ gnomeradio-1.4/src/gui.c	Tue Dec 17 23:44:10 2002
@@ -987,7 +987,6 @@
 		case GDK_KP_Subtract: 
 				gtk_adjustment_set_value(volume,(volume->value < 5) ? 0 : volume->value-5);
 				break;
-		default:
 	}
 	return FALSE;
 }
diff -urN gnomeradio-1.4.orig/src/gui.h gnomeradio-1.4/src/gui.h
--- gnomeradio-1.4.orig/src/gui.h	Tue Dec 17 23:33:03 2002
+++ gnomeradio-1.4/src/gui.h	Tue Dec 17 23:38:29 2002
@@ -69,4 +69,4 @@
 
 void change_preset(gboolean next);
 
-#endif
\ No newline at end of file
+#endif
diff -urN gnomeradio-1.4.orig/src/lirc.h gnomeradio-1.4/src/lirc.h
--- gnomeradio-1.4.orig/src/lirc.h	Tue Dec 17 23:33:03 2002
+++ gnomeradio-1.4/src/lirc.h	Tue Dec 17 23:39:17 2002
@@ -32,4 +32,4 @@
 
 void start_lirc(void);
 
-#endif
\ No newline at end of file
+#endif
diff -urN gnomeradio-1.4.orig/src/prefs.h gnomeradio-1.4/src/prefs.h
--- gnomeradio-1.4.orig/src/prefs.h	Tue Dec 17 23:33:03 2002
+++ gnomeradio-1.4/src/prefs.h	Tue Dec 17 23:41:18 2002
@@ -29,4 +29,4 @@
 
 GtkWidget* prefs_window(void);
 
-#endif
\ No newline at end of file
+#endif
diff -urN gnomeradio-1.4.orig/src/rec_tech.h gnomeradio-1.4/src/rec_tech.h
--- gnomeradio-1.4.orig/src/rec_tech.h	Tue Dec 17 23:33:03 2002
+++ gnomeradio-1.4/src/rec_tech.h	Tue Dec 17 23:38:44 2002
@@ -73,4 +73,4 @@
 int
 get_file_size(char *filename);
 
-#endif
\ No newline at end of file
+#endif
diff -urN gnomeradio-1.4.orig/src/record.h gnomeradio-1.4/src/record.h
--- gnomeradio-1.4.orig/src/record.h	Tue Dec 17 23:33:03 2002
+++ gnomeradio-1.4/src/record.h	Tue Dec 17 23:39:29 2002
@@ -27,4 +27,4 @@
 
 GtkWidget* record_status_window(void);	
 
-#endif
\ No newline at end of file
+#endif
